projects/opensearch.org/package.yml

```diff
@@ -0,0 +1,49 @@
+distributable:
+  url: https://github.com/opensearch-project/OpenSearch/archive/refs/tags/{{version}}.tar.gz
+  strip-components: 1
+
+display-name: opensearch
+
+versions:
+  github: opensearch-project/OpenSearch
+
+dependencies:
+  openjdk.org: '*'
+
+build:
+  dependencies:
+    gradle.org: '*'
+  script:
+    - gradle -Dbuild.snapshot=false ":distribution:archives:no-jdk-{{hw.platform}}-tar:assemble"
+    - mkdir -p "{{prefix}}"
+    - tar --strip-components=1 -xf distribution/archives/no-jdk-{{hw.platform}}-tar/build/distributions/opensearch-*.tar.gz -C "{{prefix}}"
+    - run: |
+        sed -i.bak 's|#\s*cluster.name: .*|cluster.name: opensearch_pkgx|' "{{prefix}}/config/opensearch.yml"
```

Comment:
since build-envs include `gnu.org/sed` you can just use `-i` instead of `-i.bak`. saves you having to clean up the backup file.